src/community/infix-short-scale.ts

```diff
@@ -0,0 +1,28 @@
+import { AbstractInfixNotation } from "./infix-abstract";
+import { toSubscript, abbreviate } from "../utils";
+import type Decimal from "break_infinity.js";
+
+// Name comes from https://en.wikipedia.org/wiki/Long_and_short_scales
+export class InfixShortScaleNotation extends AbstractInfixNotation {
+  public readonly name = "Infix short scale";
+
+  protected canHandleZeroExponent = false;
```

Comment:
Codacy found an issue: [expected member-variable-declaration: 'canHandleZeroExponent' to have a typedef](https://app.codacy.com/gh/antimatter-dimensions/notations/pullRequest?prid=6218257)